build/static_dist.js

```diff
@@ -112,12 +112,21 @@ function addPackages(siteConfig, buildFolder) {
 
 		return readFile(path.join(fullBuildFolderPath, "package.json")).then(function(packageContents){
 			var json = JSON.parse(packageContents);
+			var filePathedDeps = siteConfig.html.dependencies;
 
-			json.dependencies = _.assign(json.dependencies || {},siteConfig.html.dependencies);
+			for(var depName in siteConfig.html.dependencies) {
+				var possibleFilePath = siteConfig.dependencies[depName];
+
+				if (_.startsWith(possibleFilePath,'file:')) {
```

Comment:
I'm sorry, I don't understand what you're asking. I literally wanted to know what the code is saying: Does it start with this string or not, and lodash was already required in the file. Otherwise I would have used `indexOf`.